app/javascript/App/withLegacyCombinedProviders.tsx

```diff
@@ -2,21 +2,15 @@ import * as React from 'react'
 import CoinListContainer from './bundles/common/containers/CoinListContainer'
 import { MuiThemeProvider } from '@material-ui/core/styles'
 import theme from './theme'
-import { default as withLegacyAppReduxContainer } from './containers/app'
-import withReduxStore from './withReduxStore'
 import { DeviceProvider } from '~/bundles/common/contexts/DeviceContext'
 
 const withLegacyCombinedProviders = (TargetComponent) => {
   const WithLegacyCombinedProviders = (props, railsContext) => {
-    const TargetComponentWithRedux = withReduxStore(
-      withLegacyAppReduxContainer(TargetComponent),
-    )
-
     return (
       <DeviceProvider {...railsContext.deviceProviderProps}>
         <MuiThemeProvider theme={theme}>
           <CoinListContainer loggedIn={!!props.user}>
-            <TargetComponentWithRedux {...props} />
+            <TargetComponent {...props} />
```

Comment:
Do we need this file anymore either?

Reply:
Not entirely sure. When I did the refactor there was still a withCombinedProviders but it looks like @jessehon got rid of it in #444 
Will look into it

Reply:
Looks Like CalendarPage will lose CoinListContainer, which is provided by withLegacyCombinedProviders but not withClientProviders but might as well remove it and figure out how to reconnect it later.